app/scripts/fetch_xulrunner

```diff
@@ -483,6 +485,14 @@ if [ $BUILD_LINUX == 1 ]; then
 		if [ -e "$archived_file" ]; then
 			echo "Using $archived_file"
 			cp "$archived_file" "firefox-$GECKO_VERSION.tar.bz2"
+		elif [[ "$arch" == "aarch64" ]]; then
+			curl -O "http://ftp.de.debian.org/debian/pool/main/f/firefox-esr/firefox-esr_102.15.0esr-1~deb11u1_arm64.deb"
```

Comment:
1. Certainly would have to be 1) HTTPS and 2) not a country-specific domain
2. Can't hard-code the version number here — this could produce a broken build if config.sh is changed. This (and everything below) should embed `$GECKO_VERSION`, and if it's a 404, the script will fail and we'll know we need to find a new build.

Reply:
1. true, will change
2. currently, GECKO_VERSION is `102.15.1esr` which would not work, because the debian repo only has `102.15.0esr`. Should I create a new `GECKO_VERSION_LINUX_AARCH64` variable?